### PR TITLE
Create index.md

### PR DESCRIPTION
**Pull request by IoanaP0711**
### Pull Request Overview

This pull request adds the documentation for the project

### TODO or Help Wanted

Please let me know if there is anything that needs to be changed

### Author

Signed-off-by: Postelnecu Ioana <postelnecuioana@gmail.com>

### CODE REVIEW

File: docs/project/2024/ioana.postelnecu/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Game Console
+
+:::info 
+
+**Author**: POSTELNECU Ioana
```
Comment (alexandruradovici) [inline docs/project/2024/ioana.postelnecu/index.md:5]:
```suggestion
**Author**: POSTELNECU Ioana \
```

File: docs/project/2024/ioana.postelnecu/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# Game Console
+
```
Comment (alexandruradovici) [inline docs/project/2024/ioana.postelnecu/index.md:2]: Please add single line description of the project here.

File: docs/project/2024/ioana.postelnecu/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+## Motivation
+
+I've decided to choose this project, because ever since I was a kid, I've used to play video games and I've told myself that I'll learn how to make one from scratch. On top of that, for many years my desire for playing has been decreasing, because even though I like the games, I'm getting bored of them, so I want to create one at some point, that keeps me playing it. To sum up, I want to make this as my project, because I want to understand the basics of game development, in order to actually start designing and developing my own game. 
+
+## Architecture 
+
+This project's architecture revolves around the Raspberry Pi Pico W as the microcontroller, executing the game code. The LCD screen 1602 module serves as the display interface for the game graphics. An LED is integrated to signify the console's power status. User input is facilitated through buttons, with one controlling velocity and the other managing movement. To construct and test the circuit, a breadboard and jumper wires are employed, allowing seamless connectivity between components.
```
Comment (alexandruradovici) [inline docs/project/2024/ioana.postelnecu/index.md:20]: A diagram would be good.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill pull request template.

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

File: docs/project/2024/ioana.postelnecu/Arhitechture.JPG
Comment (alexandruradovici) [inline docs/project/2024/ioana.postelnecu/Arhitechture.JPG:1]: The spelling of the file name is incorrect :)

**Review by alexandruradovici — APPROVED** [review]

**Pull request merged**